src/Operation/Span.php

```diff
@@ -31,25 +31,29 @@ final class Span extends AbstractOperation
     public function __invoke(): Closure
     {
         return
-            /**
-             * @param callable(T, TKey, Iterator<TKey, T>): bool ...$callbacks
-             *
-             * @return Closure(Iterator<TKey, T>): Generator<int, array{0: Closure(Iterator<TKey, T>): Generator<TKey, T>, 1: array{0: Iterator<TKey, T>}}>
-             */
-            static fn (callable ...$callbacks): Closure =>
+            (
+                /**
+                 * @param array{0:(Closure(callable(T, TKey, Iterator<TKey, T>): bool ...): (Closure(Iterator<TKey, T>): Generator<TKey, T>)), 1:(Closure(callable(T, TKey, Iterator<TKey, T>): bool ...): (Closure(Iterator<TKey, T>): Generator<TKey, T>))} $operations
+                 */
+                static fn (array $operations): Closure =>
+                /**
+                 * @param callable(T, TKey, Iterator<TKey, T>): bool ...$callbacks
+                 */
+                static fn (callable ...$callbacks) =>
```

Comment:
what happened to the `Closure` return type here?

Reply:
Oops! it's fixed now.